graphql/validate.lua

```diff
@@ -245,6 +358,41 @@ local visitors = {
     }
   },
 
+  -- <argument>
+  --
+  -- {
+  --   kind = 'argument',
+  --   name = {
+  --     kind = 'name',
+  --     value = <string>,
+  --   },
+  --   value = <value>,
+  -- }
+  --
+  -- <value> is one of the following:
+  --
+  -- * <variable>
+  -- * <inputObject>
+  -- * <list>
+  -- * <enum>
+  -- * <string>
+  -- * <boolean>
+  -- * <float>
+  -- * <int>
```

Comment:
suppose, [null value](http://spec.graphql.org/October2016/#sec-Null-Value) is missing here

Reply:
It is not supported in the old lpeg based parser, which was used at times, when I wrote this patch. Now [luagraphqlparser](https://github.com/tarantool/luagraphqlparser) is there and it is supported.

So, I added it.